### ISSUE TITLE
Notebook and few questions

### ISSUE THREAD

**Issue by SebastienTs** [ctom2/noise2detail#3]
Hi, thanks for sharing this interesting approach!

I'm wondering if the notebook provided is an exact implementation of the method in the paper, and which component/variation of the algorithm each notebook cell is meant to illustrate. 

I  also wonder if you compared your method to [https://pubmed.ncbi.nlm.nih.gov/39261639](https://pubmed.ncbi.nlm.nih.gov/39261639), which superficially looks pretty similar to ZS-Noise2Noise.

Finally, do you have any straightforward strategy to extend the method to 2D+T so as to also leverage the temporal information?

**Comment by ctom2** [comment]
Hi @SebastienTs,

Thanks for the interest and the questions. Here are the clarifications:

1. Notebook Implementation: The notebook provided is meant to be an example how the denoising works on an artificial example and it follows the official implementation in this repo that has been used for evaluations in the paper. The last three cells execute the three steps described in the paper together with visualisations. 

2. Comparison to SN2N and ZS-N2N: You are correct that it shares the zero-shot philosophy and there are similarities in the way how the inputs are processed to allow for this. We did not compare Noise2Detail to the SN2N as you can see in the published paper: https://papers.miccai.org/miccai-2025/paper/3959_paper.pdf. We apply the training data subsampling from ZS-N2N, but that only makes up for one third of the whole denoising pipeline as we additionally leverage pixel shuffling to break spatial correlation of the noise and have an additional refinement round to restore details. 

3. Extending to 2D+T and leveraging temporal information is definitely feasible. For example 2D convolutions can be replaced with 3D convolutions and treating the time axis as the depth dimension. There might be simpler approaches, but it all depends on the pixel variation between the adjacent frames.

**Comment by SebastienTs** [comment]
Hi Tomáš,

Thank you for your feedback. Regarding the last point, using 3D
convolutions is often a bit aggressive for non isotropic data, which is
often the case for 2D+T data "viewed" as 3D. Would you have a more specific
way to modify your method for 2D+T fluorescence microscopy data? Is it
something you would consider for your roadmap?

Best,
Sébastien
